src/main/java/io/quarkus/gizmo2/creator/Access.java

```diff
@@ -0,0 +1,119 @@
+package io.quarkus.gizmo2.creator;
+
+import static io.github.dmlloyd.classfile.ClassFile.*;
+import static io.quarkus.gizmo2.creator.ModifierLocation.*;
+
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * The possible access levels for an item.
+ */
+public enum Access implements Modifier {
```

Comment:
I'd maybe call this `AccessLevel`, but your call.

Reply:
Sure, I can do that.